frontend/src/components/placementVisuals/AddPlacementContainer.tsx

```diff
@@ -25,31 +25,115 @@ const AddPlacementContainer = ({ onConfirm, destinationContainerList, setDestina
     const [newContainer, setNewContainer] = useState<any>({})
 
     const coordinates = useAppSelector(selectCoordinatesByID)
+    const containerKinds = useAppSelector(selectContainerKindsByID)
+
+    const getContainerKindOptions = useCallback(() => {
+      if (containerKinds) {
+        const options = Object.keys(containerKinds).filter(id => containerKinds[id].children_ids.length == 0 && id != "tube").sort().map((containerKind) => {
```

Comment:
I would do something like `containerKinds[id]?.children_ids` in case the container kind has not been loaded yet

Reply:
Also `===` is better than `==`. Javascript :)

Reply:
How can `id` be tube? Shouldn't It be an integer?

Reply:
"kinds" are still part of an older DB light version of Freezeman... They are defined in a Python file (containers.py) and do not have numerical ids like the stuff that is stored in the DB. The id is the kind name in lowercase.